bsg_test/bsg_dramsim3.cpp

```diff
@@ -213,6 +213,14 @@ extern "C" void bsg_dramsim3_tick(BSGDRAMSim3 *dramsim3)
 {
     dramsim3->clockTick();
 }
+/**
+ * Print Stats
+ * @param[in] Print stats per epoch
+ */
+extern "C" void bsg_dramsim3_print_stats(unsigned tag)
```

Comment:
unsigned int?

Reply:
using unsigned int, i get
SystemVerilog  keyword 'int' is not expected to be used in this context.

Reply:
oh wait thats just for the import dpi declaration